src/parser/paladin/holy/modules/StatValuesSpellInfo.js

```diff
@@ -25,7 +25,7 @@ export default {
   [SPELLS.LIGHT_OF_DAWN_HEAL.id]: {
     int: true,
     crit: true,
-    hasteHpct: true,
+    hasteHpct: false,
```

Comment:
Aren't you able to cast more HP generators if you have more Haste and thus cast more LoD?

Reply:
Some generators you cast for the generation (e.g. HL on beacons), others you get for spells you'd have cast regardless. Those have different value too.